quickcache/native_utc.py

```diff
@@ -0,0 +1,19 @@
+from datetime import tzinfo, timedelta
```

Comment:
I'd add a note about https://docs.python.org/2.7/library/datetime.html#tzinfo-objects in a docstring (module or class) in the spirit of attribution.